content/rdi/installation/adding-tables-to-existing-pipeline.md

```diff
@@ -114,14 +114,74 @@ The data-collections array lists tables by their fully-qualified names, using th
 
 #### Signaling Table Columns
 
-| Column | Description                                                                       |
-| ------ | --------------------------------------------------------------------------------- |
-| id     | An arbitrary string that is assigned as the identifier for the signal request.    |
-| type   | The type of signal to send.                                                       |
-| data   | An array of table names to include in the snapshot.                               |
+| Column | Description                                                                    |
+| ------ | ------------------------------------------------------------------------------ |
+| id     | An arbitrary string that is assigned as the identifier for the signal request. |
+| type   | The type of signal to send.                                                    |
+| data   | An array of table names to include in the snapshot.                            |
 
 ## SQL Server: `sp_cdc_enable_table` Stored Procedure Arguments
 
 - `@source_name` - Specifies the name of the table that you want to capture.
 - `@role_name` - Specifies a role MyRole to which you can add users to whom you want to grant SELECT permission on the captured columns of the source table. Users in the sysadmin or db_owner role also have access to the specified change tables. Set the value of @role_name to NULL, to allow only members in the sysadmin or db_owner to have full access to captured information.
 - `@filegroup_name` - Specifies the filegroup where SQL Server places the change table for the captured table. The named filegroup must be already exist. It is best not to locate change tables in the same filegroup that you use for source tables.
+
+## Example for Adding a Signaling Table for Oracle Database
+
+1. Create a signaling table `DEBEZIUM_SIGNAL`:
+
+   ```sql
+   CREATE TABLE DEBEZIUM_SIGNAL
+   (
+   id VARCHAR(42) PRIMARY KEY,
+   type VARCHAR(32) NOT NULL,
+   data VARCHAR(2048) NULL
+   );
+   ```
+
+2. Add the property `debezium.source.signal.data.collection` to the `application.properties` file:
```

Comment:
This is outdated, you should explain how to add this to the `advanced` section, no more `application.properties`.